pkg/build/api/types.go

```diff
@@ -42,8 +49,8 @@ type Build struct {
 // BuildSpec encapsulates all the inputs necessary to represent a build.
 type BuildSpec struct {
 	// ServiceAccount is the name of the ServiceAccount to use to run the pod
-	// created by this build.
-	// The pod will be allowed to use secrets referenced by the ServiceAccount
+	// created by this build. The pod will be allowed to use secrets referenced
```

Comment:
@rhcarvalho i'm of course aware of the "small commits" "theory" and I don't think it always works well as it is advertised.
Github UI, when you do comment on particular commit instead of doing comments at Files tab, what you will discover is that when the person force push, all the commit comments will suddenly disappear. If you do comment on Files tab, they will stay there as "outdated diff".

Again:

> But our single-commit approach and you comment about "unrelated changes" push us against making trivial improvements.

This was not a trivial change and i'm not pushing anyone from doing "trivial" improvements if they are trivial.

> This exaggeration is not funny ;-)
> I'm not talking about one character change per commit nor anything alike.

I never said/wrote this, so who is exaggerating now? I wrote, that doing multiple commits makes it harder to review because you have to track the changes between commits (you add something in commit 1 and remove it in commit 2) and you have to review each commit separately because the Files tab will show you all changes (including the "trivial" and "cosmetic" changes).